### DISCUSSION TITLE
Make It Easier to Toggle Between Windows Ink and Aboslute Mode

### DISCUSSION THREAD

**Discussion by ghost** [OpenTabletDriver/OpenTabletDriver#2382] [Ideas]
In order to draw I have to set up 3 things:

1. change tablet area to full area
2. activate Windows Ink Absolute Mode
3. go to Pen Settings and change Tip Binding to Windows Ink

In Hawkus TabletDriver you didn't need to do the 3rd step (if I'm not mistaken the existence of the 3rd step is windows fault) and on Wacom Drivers you can create profiles ultimately removing every step.
Is there a possiblity you can simplifiy this?

**Comment by jamesbt365** [comment]
As this mode is from a plugin this will likely not get much easier than just saving a preset which is a feature. Click `file` in the top of otd and you'll see the options for it.

**Reply by InfinityGhost** [reply]
As for future versions, you will be able to set up a binding to switch profiles as well. (In #2352)
